packages/xo-server-perf-alert/package.json

```
@@ -2,7 +2,7 @@
   "name": "xo-server-perf-alert",
   "version": "0.2.0",
   "license": "AGPL-3.0",
-  "description": "",
+  "description": "The alert plugin for Xo-Server",
```

Example of what could be said:

```suggestion
  "description": "Sends alerts based on performance criteria",
```